From a23d97852bd5a6693920a4f9c823f606e244f783 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Thu, 14 Mar 2019 12:46:38 +0000
Subject: [PATCH 10/13] Check for UEFI EDK2 firmware on ARMv7

Checks for EDK2 firmware on ARMv7 along side aarch64 as it now also
supports UEFI now if the firmware is available.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 oz/Guest.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/oz/Guest.py b/oz/Guest.py
index 3bb144e..fc3ae6d 100644
--- a/oz/Guest.py
+++ b/oz/Guest.py
@@ -484,7 +484,7 @@ class Guest(object):
             oz.ozutil.lxml_subelement(osNode, "initrd", initrd)
         if cmdline:
             oz.ozutil.lxml_subelement(osNode, "cmdline", cmdline)
-        if self.tdl.arch == "aarch64":
+        if self.tdl.arch in ["aarch64", "armv7l"]:
             loader, nvram = oz.ozutil.find_uefi_firmware(self.tdl.arch)
             oz.ozutil.lxml_subelement(osNode, "loader", loader, {'readonly': 'yes', 'type': 'pflash'})
             oz.ozutil.lxml_subelement(osNode, "nvram", None, {'template': nvram})
-- 
2.20.1

